### PR TITLE
Document `fastboot.method` and `fastboot.body`

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: markdown/docs/user-guide.md [hunk 1]
```diff
@@ -257,6 +257,96 @@ export default Ember.Route.extend({
...
+On `POST`, `PUT` and `PATCH` request you will probably be interested in the body of the request.
+You can access the `body` of the request via `fastboot.request` in the `fastboot` service, but
+only of you setup a middleware to extract the body of the request in your express server.
+
+You can use `body-parser` and `fastboot-express-middleware` and to create an in-repo addon that
+contains a middleware.
+
+
+```javascript
+// your-app/lib/fastboot-middleware/index.js
```
Comment (kratiahuja) [inline markdown/docs/user-guide.md:287]: Do you mean to have this middleware for dev or production?

**Review by kratiahuja — COMMENTED** [review]
LGTM overall. Except for the question below.

File: markdown/docs/user-guide.md (hunk 1 above)
Comment (cibernox) [reply markdown/docs/user-guide.md:287]: Ideally both. I like to have my dev env respond to the same kind of requests I receive in prod. What changes would you add to make this "universal"?

Comment (kratiahuja) [reply markdown/docs/user-guide.md:287]: > What changes would you add to make this "universal"?

1. This should be a seperate middleware so people can add as needed.
2. It should check if it is the index.html request and then call fastbootExpressMiddleware

For the record, I am fine with merging this as long as you update it to mention the above.

Comment (cibernox) [reply markdown/docs/user-guide.md:287]: I *think* I've applied the feedback, let me know if I misunderstood something.

**Pull request merged**